On nixos x86_64:

```builder for '/nix/store/0qqrw8h5d6rcfnpx0d6kf8ayxhq1pim8-tuntox-0.0.10.drv' failed with exit code 2; last 10 log lines:
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/client.c:438: undefined reference to `tox_friend_send_lossless_packet'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/client.c:351: undefined reference to `tox_friend_add'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/client.c:376: undefined reference to `tox_friend_get_connection_status'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/client.c:399: undefined reference to `tox_friend_delete'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: gitversion.o: in function `print_version':
  /build/source/gitversion.c:8: undefined reference to `tox_version_patch'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/gitversion.c:8: undefined reference to `tox_version_minor'
  /nix/store/x2zbyl2l4xayv4d5is0b31plfq8xjjym-binutils-2.35.2/bin/ld: /build/source/gitversion.c:8: undefined reference to `tox_version_major'
  collect2: error: ld returned 1 exit status
  make: *** [Makefile:35: tuntox] Error 1
```

--- a/pkgs/development/libraries/libtoxcore/default.nix
+++ b/pkgs/development/libraries/libtoxcore/default.nix
@@ -53,7 +53,7 @@ in {
   };
 
   libtoxcore_0_2 = generic {
-    version = "0.2.13";
-    sha256 = "0a1cp00bnxl3q4l74yqp4aa6fg9slz4rg4lfzkl3khvmm6nzckds";
+    version = "0.2.17";
+    sha256 = "sha256-SOI6QKOSt/EK9JDrSaV6CrD5sx8aYb5ZL3StYq8u/Dg=";
   };
 }

--- a/pkgs/tools/networking/tuntox/default.nix
+++ b/pkgs/tools/networking/tuntox/default.nix
@@ -9,6 +9,7 @@
 , libsodium
 , libtoxcore
 , libvpx
+, msgpack
 , pkg-config
 , python3
 , python3Packages
@@ -27,7 +28,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cscope git pkg-config ];
 
-  buildInputs = [ libopus libtoxcore libsodium libevent libvpx python3 ];
+  buildInputs = [ libopus libtoxcore libsodium libevent libvpx msgpack python3 ];
 
   pythonBuildInputs = with python3Packages; [
     jinja2

